### DISCUSSION TITLE
Github sponsorship

### DISCUSSION THREAD

**Discussion by arcanetrivia** [AprilSylph/XKit-Rewritten#657] [Lounge]
Just curious why you withdrew from the sponsorship program? I know this is a lot of work to make and since I'm on Tumblr every day, that means I'm using Xkit every day. Even if it was only a few bucks I liked recognizing that financially.

**Comment by AprilSylph** [comment]
It was required as part of my exclusivity agreement with Automattic; I'm now a full-time employee, working on Tumblr itself.

**Comment by arcanetrivia** [comment]
Ahhh. Neat!
